src/main/java/io/stargate/sgv2/jsonapi/config/constants/VectorConstant.java

```diff
@@ -13,17 +13,17 @@ public interface VectorConstant {
       Map.of(
```

Comment:
nit: should probably be `SUPPORTED_SOURCE_MODELS` (instead of `SUPPORTED_SOURCE_MODELS`).